dev/devicelab/bin/tasks/cubic_bezier_perf__e2e_summary.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_devicelab/tasks/perf_tests.dart';
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+
+// TODO(CareF): add sksl warmup version
```

Comment:
As you've already added the SkSL version for gallery, I wonder if it's easy to add the SkSL version for cubic_bezier_perf in this PR so we can have 1 less TODO. It will our tech debt :D

Reply:
Done. Tested against my Xiaomi MIX and seems normal.